README.md

```diff
@@ -67,6 +67,10 @@ If your subscriptions list doesn't contain `Azure SQL DB Project Orcas - CitusDa
     ssh-add
     ```
 
+5. You should setup your VPN to be able to connect to Azure VM-s if your tests are not running on CircleCI. Doing this as of lastest consists of:
```

Comment:
iirc circle ci does also suffer from the same problem, what should we do about that ?

Also, nit:

```suggestion
5. You should setup your VPN to be able to connect to Azure VM-s if your tests are not running on CircleCI. Doing this as of latest consists of:
```

Reply:
I am not sure why the issue happens with circle CI. The problem with Azure is due to them changing the rules of connecting to VM-s with ssh.